nix/docker.nix

```diff
@@ -68,9 +68,11 @@ let
       curl # CLI tool for transferring files via URLs
       env-shim # Make /usr/bin/env available
       findutils # GNU find
+      getconf # get num cpus
```

Comment:
It doesn't seem like this is used?

Reply:
Had to include it as the [restore script](https://github.com/input-output-hk/cardano-db-sync/blob/035d30b01850df97915063e1eb68f6e24efd7ddc/scripts/postgresql-setup.sh#L28) requires checking cores